app/src/main/res/layout/item_rolled_dice.xml

```diff
@@ -6,53 +6,53 @@
     android:layout_height="wrap_content"
     android:paddingVertical="4dp">
 
-
     <ImageView
-        android:id="@+id/rolledDiceIcon"
-        android:layout_width="64dp"
-        android:layout_height="64dp"
+        android:id="@+id/rolled_dice_icon"
+        android:layout_width="@dimen/rolled_die_image_size"
+        android:layout_height="@dimen/rolled_die_image_size"
+        android:contentDescription="@string/icon_dice_type"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:srcCompat="@drawable/nav_menu_d20_icon"
-        app:tint="@color/white"
-        android:contentDescription="@string/icon_dice_type" />
+        app:tint="@color/white" />
 
     <TextView
-        android:id="@+id/rolledDiceText"
-        android:textStyle="bold"
-        android:textSize="16sp"
+        android:id="@+id/rolled_dice_text"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="16dp"
+        android:layout_marginStart="@dimen/view_vertical_margin_medium"
         android:layout_marginBottom="4dp"
-        android:text="@string/rolled_dice_text"
         android:textColor="@color/white"
+        android:textSize="@dimen/roll_history_text_size"
+        android:textStyle="bold"
         app:layout_constraintBottom_toTopOf="@+id/divider"
-        app:layout_constraintStart_toEndOf="@+id/rolledDiceIcon" />
+        app:layout_constraintStart_toEndOf="@+id/rolled_dice_icon"
+        tools:text="You rolled a nat 20!" />
 
     <View
         android:id="@+id/divider"
-        android:layout_width="409dp"
```

Comment:
‼️ Measurements in DP should always be divisible by 4. This is a convention in android because it supports many screen sizes. Something divisible by 2 is allowed in a tight spot.